Updates index.html based on #278 proposal (#296)
sa-tre/satre-specification: docs/source/index.md

contributing/contributors.md
```

(what_is_satre)=

## 👀 What is SATRE?

The SATRE project provides a Standard Architecture for {ref}`Trusted Research Environments (TREs) <what_tre>`. 
It incorporates knowledge and best practices from multiple institutions and sectors across the UK. 
This includes all aspects of TRE provision such as information governance procedures, computing technology, data management and other capabilities.

It aims to standardise the capabilities of TREs, making it easier for users, operators, and developers to work with sensitive data, and making the operation of TREs more transparent to data owners and the general public.

This specification should be useful if you are:

> - a {ref}`TRE Operator <infrastructure_roles>` wanting to evaluate or improve their TRE with the suggested capabilities.
> - a {ref}`Developer or Builder <infrastructure_roles>` of new TREs looking for guidance in their thinking and decision making.

Reviewer: ```suggestion
- a {ref}`TRE Operator <infrastructure_roles>` wanting to evaluate or improve their TRE with the suggested capabilities
- a {ref}`Developer or Builder <infrastructure_roles>` of new TREs looking for guidance in their thinking and decision making
```

<!-- Motivation: Why a TRE specification is needed/ useful and a description of the broader SATRE project, conception and goals -->

The need for trusted research environments (TREs) is clear. 
Influential reports including the UK Government's [Goldacre review](https://www.gov.uk/government/publications/better-broader-safer-using-health-data-for-research-and-analysis) and [‘Data Saves Lives’](https://www.gov.uk/government/publications/data-saves-lives-reshaping-health-and-social-care-with-data/data-saves-lives-reshaping-health-and-social-care-with-data) policy paper, have  highlighted the need for change in how sensitive data are handled. 
These papers set out a vision for the potential impact of research enabled by TREs.

 At present operators have to interpret a range of frameworks, legislation and guidance when building and running a TRE. 
  These include:
 - [Office for National Statistics: 5 Safes](https://blog.ons.gov.uk/2017/01/27/the-five-safes-data-privacy-at-ons/)
 - [UK Health Data Research Alliance: TRE Green Paper](https://zenodo.org/record/4594704#.Yd8RRdHP1zq)
 - [UK Health Data Research Alliance: TRE Principles and Best Practices](https://zenodo.org/record/5767586)

>  - [Design choices for productive, secure, data-intensive research at scale in the cloud](https://arxiv.org/pdf/1908.08737.pdf)

Reviewer: ```suggestion
 - [Design choices for productive, secure, data-intensive research at scale in the cloud](https://arxiv.org/abs/1908.08737)
```

 - [ISO27001](https://www.iso.org/standard/27001)
 - [Digital Economy Act](https://www.legislation.gov.uk/ukpga/2017/30/contents/enacted)
 - [Handbook on Statistical Disclosure Control for Outputs](https://ukdataservice.ac.uk/app/uploads/thf_datareport_aw_web.pdf)
 
This makes for inconsistent governance standards and makes it hard for researchers to work consistently in different environments.

A common specification for TREs will improve governance and practice across the sector, simplify researcher and operator journeys. 

> Furthermore it will lay a foundation for interoperability that is required to maximise the impact of research by providing a trusted ecosystem for providing access to currently disparate and siloed data.

Reviewer: ```suggestion
Furthermore, it will lay a foundation for interoperability that is required to maximise the impact of research by providing a trusted ecosystem for working with currently disparate and siloed data.
```

This sentence is quite long and complex. It would be worth breaking it down.
